ansible/README.md

```diff
@@ -249,6 +253,27 @@ In case one or more tasks fail or should not be run in the local environment, se
 ```bash
 ansible-playbook -b AdoptOpenJDK_Unix_Playbook/main.yml --skip-tags="install_zulu,jenkins_authorized_key,nagios_add_key,add_zeus_user_key"
 ```
+
+## Executing under vagrant (Windows)
+
+Ansible cannot be installed on a Windows machine, so you should boot a Linux VM using Vagrant and install it there instead:
+1) Copy a Linux Vagrantfile from  the `openjdk-infrastructure/ansible/vagrant` directory into the `openjdk-infrastructure/ansible` directory, and save it without an extension.
```

Comment:
If you can show the example of this using powershell commands that would be a bonus:

e.g.

```powershell
copy ./openjdk-infrastructure/ansible/vagrant/Vagrantfile.Ubuntu14 ./openjdk-infrastructure/ansible
```

Reply:
I'll add that example and also an example of renaming to remove the extension

Reply:
Make sure the slashes are the appropriate way round e.g. `\` if using the Windows `copy` command and `/` if using `cp`